launch-cxx.in

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Should these files be in the root dir?

Reply:
yes, unless you can point me the better dir for these files

Reply:
I'm not sure that I understand their purposes

Reply:
look at ccache, there is a link